docs/api.md

```diff
@@ -1,1300 +1,1203 @@
-
-<details>
-  <summary>Response</summary>
-
+**Response (No wallet - 404):**
 ```json
 {
-  "wallet": {
-    "address": "0xAafd5cb31a611C5e5aa65ea8c6226EB4328175E1",
-    "safe_chains": [],
-    "ledger_type": 0,
-    "safes": {},
-    "safe_nonce": null
-  },
-  "mnemonic": ["polar", "mail", "tattoo", "write", "track", ... ]
+  "error": "Please create a wallet first"
 }
 ```
 
-</details>
-
----
-
-### `POST /api/wallet/safe`
-
-Creates a Gnosis safe for given chain.
-
-<details>
-  <summary>Request</summary>
-
-Using default new safe funding:
-
+**Response (No safe - 404):**
 ```json
 {
-  "chain": "gnosis",
-  "backup_owner": "0x46eC2E77Fe3E367252f1A8a77470CE8eEd2A985b"
+  "error": "Please create a safe first for this chain"
 }
 ```
 
-Specify new safe funding:
+### `POST /api/wallet/safe`
+
+Create a new Gnosis Safe.
 
+**Request Body:**
 ```json
 {
   "chain": "gnosis",
-  "backup_owner": "0x46eC2E77Fe3E367252f1A8a77470CE8eEd2A985b",
+  "backup_owner": "0x...",
   "initial_funds": {
-    "0x0000000000000000000000000000000000000000": 1000000000000000000,
-    "0xcE11e14225575945b8E6Dc0D4F2dD4C570f79d9f": 0
+    "0x0000000000000000000000000000000000000000": "1000000000000000000"
   }
 }
 ```
 
-Transfer all excess assets (except operational Master EOA balance) to the new safe:
-
+**Request Body (with asset transfer):**
 ```json
 {
-  "chain": "gnosis",
-  "backup_owner": "0x46eC2E77Fe3E367252f1A8a77470CE8eEd2A985b",
-  "transfer_excess_assets": true
+  "chain": "gnosis", 
+  "backup_owner": "0x...",
+  "transfer_excess_assets": "true"
 }
 ```
 
-</details>
-
-<details>
-  <summary>Response</summary>
+**Response (Success - 201):**
+```json
+{
+  "create_tx": "0x...",
+  "transfer_txs": {
+    "0x0000000000000000000000000000000000000000": "0x..."
+  },
+  "safe": "0x...",
+  "message": "Safe created successfully"
+}
+```
 
-- If Gnosis safe creation is successful:
+**Response (Safe exists - 409):**
+```json
+{
+  "safe": "0x...",
+  "message": "Safe already exists for gnosis"
+}
+```
 
-  ```json
-  {
-    "create_tx": "0xac14dcd5938c71cd97388307c41477dc2f2a4e97b6b2641cef123a769898bd03",
-    "transfer_txs": {
-      "0x0000000000000000000000000000000000000000": "0x0036489a4a27b4ee1b7e4a37ae5597b895bf43e4cda25c7e6c0f1d02f8c098aa",
-      "0xcE11e14225575945b8E6Dc0D4F2dD4C570f79d9f": "0x45304084049916535419d4b175074cdfcaeea59e3a5d34010854b3ac049261b2"
-    },
-    "safe": "0x29e23F7705d849F368855947691cB133CD770752",
-    "message": "Safe created!"
-  }
-  ```
+**Response (Invalid request - 400):**
+```json
+{
+  "error": "Please specify either 'initial_funds' or 'transfer_excess_assets', but not both."
+}
+```
 
-- If Gnosis safe creation is not successful:
+**Response (No wallet - 400):**
+```json
+{
+  "error": "Please create a wallet first"
+}
+```
 
-  ```json
-  {
-    "error": "Error message",
-    "traceback": "Traceback message"
-  }
-  ```
+**Response (Not logged in - 401):**
+```json
+{
+  "error": "Please log in to create a safe"
```

Comment:
Error should describe what the issue is: "User not logged in", but not ask to do it.